docs/src/content/docs/victory-pie.md

```diff
@@ -519,6 +519,61 @@ See the [Data Accessors Guide][] for more detail on formatting and processing da
 y={(d) => d.value + d.error}
 ```
 
+## labelIndicator
+
+`type: boolean || element`
+
+The `labelIndicator` prop defines the label indicator line between labels and the pie chart. If this prop is used as a boolean,then the default indicator will be displayed. To customize or pass your own styling `<LineSegment/>` can be passed to labelIndicator. LabelIndicator is functional only when labelPosition = "centroid". To adjust the labelIndicator length, `labelIndicatorInnerOffset` and `labelIndicatorOuterOffset` props can be used alongside labelIndicator. 
+
+```playground
+<VictoryPie
+  data={sampleData}
+  labelIndicator 
+  style={{ labels: { fill: "white", fontSize: 20, fontWeight: "bold" } }}
+/>
+<VictoryPie
+  data={sampleData}
+  labelIndicator={<LineSegment style = {{stroke:"red", strokeDasharray:1,fill: "none",}}/>}
+  style={{ labels: { fill: "white", fontSize: 20, fontWeight: "bold" } }}
+/>
+<VictoryPie
+  data={sampleData}
+  labelIndicator={<LineSegment style = {{stroke:"red", strokeDasharray:1,fill: "none",}}/>}
+  style={{ labels: { fill: "white", fontSize: 20, fontWeight: "bold" } }}
+  labelIndicatorInnerOffset = {10}
+  labelIndicatorOuterOffset = {15}
```

Comment:
```suggestion
  labelIndicatorInnerOffset={10}
  labelIndicatorOuterOffset={15}
```

Minor thing, but for consistency we should remove the spaces between the prop name and value

Reply:
updated it